pip/commands/install.py

```diff
@@ -213,6 +196,12 @@ def run(self, options, args):
             )
 
         if options.download_dir:
+            warnings.warn(
+                "pip install --download has been deprecated and will be "
+                "removed in the future. Pip now has a download command that "
+                "should be used instead.",
+                RemovedInPip8Warning,
```

Comment:
Yea it should be ``RemovedInPip10Warning``. We have the versioned names to make it easier, but essentially the first release it's a ``PendingDeprecation`` warning, when the next major version bump happens it'll switch to a ``DeprecationWarning``, and when the next major version bump happens it gets removed. That's all abstracted by the ``RemovedInPipXXWarning``s though, generally you just want to use whatever the highest version is of that.